### ISSUE TITLE
Elevation Band File

### ISSUE THREAD

**Issue by Saadi4469** [lokendrarathore/vic-auto#12]
Hi Sir,

The script ended up working fine, but I have another question, based on the following sample lines from the file:

[Elevation_Band_File_test.txt](https://github.com/LSRathore/vic-auto/files/5496139/Elevation_Band_File_test.txt)
 
As you can see the grid cells right after the first grid cell show fractional area zero for Band 1 (this means that this band does not exist in those grid cells), that means actually that the next band which is band 2 should be band 1. So, should I delete bands from grid cells which have fractional area of zero?

Thank you,

**Comment by LSRathore** [comment]
No, let it the way as it is because if the fractional area is zero for any band then the model does not perform any calculation for it. If you delete the band 1 and assign band 2 as 1, then model will assume range of band 1 for band 2 and it would do wrong calculation. So do not delete any band if its the fraction is zero.

**Comment by Saadi4469** [comment]
Thank for you further clarification sir!!
